sound/soc/intel/boards/sof_rt1308.c

```diff
@@ -178,16 +286,96 @@ static struct snd_soc_dai_link *sof_card_dai_links_create(struct device *dev,
 		id++;
 	}
 
+	/* HDMI-Out SSP */
```

Comment:
no, this is not based on SSP (unlike HDMI-in).

Suggest renaming as /* HDMI playback */

Reply:
fixed